src/main/java/dinkplugin/util/InstantAdapter.java

```diff
@@ -8,6 +8,13 @@
 import java.time.Instant;
 import java.time.format.DateTimeParseException;
 
+/**
+ * Serialized and deserializes {@link Instant} instances
+ * from their equivalent ISO-8601 string representation.
+ * <p>
+ * Discord requires this string format, rather than epoch milliseconds
+ * that the default RuneLite Gson instance would produce.
```

Comment:
To be consistent with our own documentation

```suggestion
 * that the default RuneLite GSON instance would produce.
```